enhancements/network/ovn-hypershift-live-migration.md

```diff
@@ -0,0 +1,303 @@
+---
+title: ovn-hypershift-live-migration
+authors:
+  - "@ellorent"
+reviewers:
+  - "@martinkennelly"
+  - "@dcbw"
+  - "@danwinship"
+  - "@tssurya"
+  - "@numans"
+approvers:
+  - "@martinkennelly"
+api-approvers:
+  - "@martinkennelly"
+creation-date: 2023-02-06
+last-updated: 2023-02-06
+tracking-link:
+  - "https://issues.redhat.com/browse/CNV-22946"
+see-also:
+  - "https://hypershift-docs.netlify.app/"
+---
+
+
+# OVN-kubernetes live migration for hypershift kubevirt provider
+
+## Summary
+
+The [Hypershift](https://hypershift-docs.netlify.app/) project creates openshift clusters on demand on top of already provisioned openshift by deploying  (and attaching working nodes to) "Hosted Control Planes" using one of different providers, such as aws, azure, baremetal (agent), and kubevirt.
+
+In KubeVirt's case, the worker nodes will be spun as KubeVirt virtual machines.
+
+There are some network requirements, some of them are imposed by kubevirt 
+live migration and others by kubernetes:
+
+- The IP Kubelet uses to register/communicate should follow the node during live migration.
+- Guest Cluster Network isolation. Guest clusters shouldn’t be able to talk to other guest clusters directly except through public LBs. Guest clusters shouldn’t be able to talk to any infra components except through public LBs.
+- No TCP connection breakage over E/W and N/S during live migration with minimal disruption.
+- Support exposing workers via services. 
+- Allow workers services access to services like kubedns and apiserver from hosted control plane.
+
+To accomplish the goals listed above the ovn-kubernetes default network has to implement some features triggered for an Hypershift KubeVirt VM.
+hypershift kubevirt VM is detected.
+
+## Motivation
+
+To reach GA at hypershift kubevirt provider it has to implement live migration and adding live-migration features to ovn-k default pod network
```

Comment:
Hypershift and kubevirt integration requires the ability to have live-migration functionality across the network with OVN-Kubernetes.